packages/yew/src/virtual_dom/vnode.rs

```diff
@@ -204,7 +204,7 @@ impl fmt::Debug for VNode {
 impl PartialEq for VNode {
     fn eq(&self, other: &VNode) -> bool {
         match (self, other) {
-            (VNode::VTag(a), VNode::VTag(b)) => a == b,
+            (VNode::VTag(a), VNode::VTag(b)) => a.eq(b),
```

Comment:
What is the difference  in behaviour here?

Reply:
Hmm,  this is mismerged. I'll run over the diff again.